site2/website/release-notes.md

```diff
@@ -1,6 +1,181 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.7.1 &mdash; 2021-03-18 <a id=“2.7.1”></a>
+
+#### Broker
+
+- Fix topic ownership is not checked when getting topic policy [#9781](https://github.com/apache/pulsar/pull/9781)
+- Fix the issue of consumers cannot be created for older subscriptions if the limit of `maxSubscriptionsPerTopic` is reached [#9758](https://github.com/apache/pulsar/pull/9758)
+- Fix marking individual deletes as dirty [#9732](https://github.com/apache/pulsar/pull/9732)
+- Don't add broker-address header if response has already been committed [#9744](https://github.com/apache/pulsar/pull/9744)
```

Comment:
If it's a fix, maybe we can describe it in another way, such as
Add broker address header name and check the commit response before adding a header.
You can consult Yu and make a decision whether to refine it.